app/views/recommendations/edit.html.erb

```diff
@@ -2,8 +2,6 @@
   Review form - <%= t('page_title') %>
 <% end %>
 
-<% content_for :title, "Review assessment" %>
```

Comment:
- Removed title as it wasn't being used
  - I'm having to change text which is 'Review assessment' but
    I don't see where this is used so I've removed it